src/accelerate/utils/dataclasses.py

```diff
@@ -968,6 +975,22 @@ def _deepspeed_config_checks(self):
                 "It will only ask for the necessary config variables when using `deepspeed_config_file`."
             )
 
+    def set_moe_leaf_modules(self, model):
+        from deepspeed.utils import set_z3_leaf_modules
+
+        if self.transformer_moe_cls_names is None:
+            self.transformer_moe_cls_names = os.environ.get("ACCELERATE_DEEPSPEED_MOE_LAYER_CLS_NAMES", None)
```

Comment:
Does `ACCELERATE_DEEPSPEED_MOE_LAYER_CLS_NAMES` need to be documented in the section on env vars?

Reply:
Done. the cmd argument for this is now documented.